instrumentation/graphql/test/instrumentation/graphql/tracers/graphql_tracing_test.rb

```diff
@@ -42,20 +40,11 @@
     it 'traces platform keys' do
       result = SomeGraphQLAppSchema.execute(query_string, variables: { id: 1 })
 
-      graphql_tracer.platform_keys.each do |_key, value|
```

Comment:
I had two remove the two usages of `graphql_tracer.platform_keys` because there was no easy way to dynamically get the tracer which was installed. That's one downside of using the same test file.